blog.html

```diff
@@ -22,6 +22,46 @@
         <!--#include virtual="includes/_nav.htm" -->
         <div class="right">
             <h1 class="content-title">Blog</h1>
+            <article>
+                <h2 class="bullet">
+                    <a id="apache_kafka_360_release_announcement"></a>
+                    <a href="#apache_kafka_360_release_announcement">Apache Kafka 3.6.0 Release Announcement</a>
+                </h2>
+                15 Sep 2023 - Satish Duggana (<a href="https://twitter.com/0xeed">@SatishDuggana</a>)
+                <p>We are proud to announce the release of Apache Kafka 3.6.0. This release contains many new features and improvements. This blog post will highlight some of the more prominent features. For a full list of changes, be sure to check the <a href="https://downloads.apache.org/kafka/3.6.0/RELEASE_NOTES.html">release notes</a>.</p>
+                <p>See the <a href="https://kafka.apache.org/36/documentation.html#upgrade_3_6_0">Upgrading to 3.6.0 from any version 0.8.x through 3.5.x</a> section in the documentation for the list of notable changes and detailed upgrade steps.</p>
+                <p>The ability to migrate Kafka clusters from ZK to KRaft mode with no downtime is still an early access feature. It is currently only suitable for testing in non production environments. See <a href="https://cwiki.apache.org/confluence/display/KAFKA/KIP-866+ZooKeeper+to+KRaft+Migration">KIP-866</a> for more details.</p>
```

Comment:
```suggestion
                <p>
                    The ability to migrate Kafka clusters from a ZooKeeper metadata system to a KRaft metadata system is
                    now considered stable and suitable for production environments. See the ZooKeeper to KRaft migration
                    <a href="https://kafka.apache.org/documentation/#kraft_zk_migration">operations documentation</a> for
                    details. Note that support for JBOD is still not available for KRaft clusters, therefor clusters
                    utilizing JBOD cannot be migrated. See <a href="https://cwiki.apache.org/confluence/display/KAFKA/KIP-858%3A+Handle+JBOD+broker+disk+failure+in+KRaft">KIP-858</a>
                    for details regarding KRaft and JBOD.
                </p>
```

Here's what I had in my other PR, let's just include it in this one.